governance/pyth_staking_sdk/src/constants.ts

```diff
@@ -0,0 +1,18 @@
+import { PublicKey } from "@solana/web3.js";
+
+export const EPOCH_DURATION = 60n * 60n * 24n * 7n; // 1 week
+
+export const POSITION_BUFFER_SIZE = 200;
+export const POSITIONS_ACCOUNT_SIZE = 8 + 32;
+
+export const STAKING_PROGRAM_ADDRESS = new PublicKey(
+  "pytS9TjG1qyAZypk7n8rw8gfW9sUaqqYyMhJQ4E7JCQ"
```

Comment:
I think the IDLs have these addresses already, see if you can get them from there

Reply:
let's deal with this later